parts/k8s/addons/secrets-store-csi-driver.yaml

```diff
@@ -282,6 +282,18 @@ status:
   conditions: []
   storedVersions: []
 ---
+# A priority class for the daemonset such that they are not
```

Comment:
Could you use go template comments for these lines instead of bash comments to save cloud-init data overhead? :/

E.g.:

```
{{/* A priority class for the daemonset such that they are not */}}
```

Reply:
Ok - annoying, but ok.
(We could look loading and dumping yaml in optimal form after templating such that you are sure to not have waste in general)